plugins/toolkit/jetbrains-core/src-241+/software/aws/toolkits/jetbrains/core/explorer/ConnectionActionToolbarBuilder.kt

```diff
@@ -0,0 +1,35 @@
+// Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package software.aws.toolkits.jetbrains.core.explorer
+
+import com.intellij.openapi.actionSystem.ActionManager
+import com.intellij.openapi.actionSystem.ActionPlaces
+import com.intellij.openapi.actionSystem.ActionToolbar
+import com.intellij.openapi.actionSystem.DefaultActionGroup
+import com.intellij.openapi.actionSystem.toolbarLayout.ToolbarLayoutStrategy
+import com.intellij.util.ui.components.BorderLayoutPanel
+
+object ConnectionActionToolbarBuilder {
+    fun createToolbar(toolWindow: AwsToolkitExplorerToolWindow, group: DefaultActionGroup): BorderLayoutPanel {
+        return BorderLayoutPanel().apply {
+            addToCenter(
+                ActionManager.getInstance().createActionToolbar(ActionPlaces.TOOLBAR, group, true).apply {
+                    layoutStrategy = ToolbarLayoutStrategy.NOWRAP_STRATEGY
+                    setTargetComponent(toolWindow)
```

Comment:
Since only this section differs across the versions, can we move the creation of this component to the util file and retain the creation for the panel in the init block of AWSToolkitExplorerToolWindow?

Reply:
Done & done, Thanks for the feedback :)